src/actions/storage.action.ts

```diff
@@ -490,4 +515,183 @@ export const storageActionsThunk = {
       dispatch(NotificationActions.addError(result.error!, result.cause));
     }
   },
+
+  fetchSavedRegisteredKeymaps: (
+    definitionId: string
+  ): ThunkPromiseAction<void> => async (
+    // eslint-disable-next-line no-unused-vars
+    dispatch: ThunkDispatch<RootState, undefined, ActionTypes>,
+    // eslint-disable-next-line no-unused-vars
+    getState: () => RootState
+  ) => {
+    const { storage } = getState();
+    const resultList = await storage.instance!.fetchSavedRegisteredKeymaps(
+      definitionId
+    );
+
+    if (resultList.success) {
+      dispatch(
+        StorageActions.updateSavedRegisteredKeymapList(
+          resultList.savedKeymaps as SavedRegisteredKeymapData[]
+        )
+      );
+    } else {
+      console.error(resultList.cause);
```

Comment:
@adamrocker In this case, it does not only output a debug log with `console.log`, but it should also show the error notification (and to send a error report via Error Reporting API).